app/views/auth/registrations/_sessions.html.haml

```diff
@@ -0,0 +1,16 @@
+%h6= t 'sessions.title'
+%p.muted-hint= t 'sessions.explanation'
+
+%table.table
+  %thead
+    %tr
+      %th= t 'sessions.browser'
+      %th= t 'sessions.ip'
+      %th= t 'sessions.activity'
+  %tbody
+    - @sessions.each do |session|
+      %tr
+        %td= t 'sessions.description', browser: session.browser, platform: session.platform.capitalize
```

Comment:
* `session.browser` is not localized (e.g. "Generic Browser")
* `session.platform` is derived from ID and may not be human reable.